third_party/docfx/templates/devsite/partials/class.header.tmpl.partial

```diff
@@ -120,7 +120,7 @@
 <div class="markdown level0 remarks">{{{remarks}}}</div>
 {{/remarks}}
 {{#example.0}}
-<h5 id="{{id}}_examples"><strong>{{__global.examples}}</strong></h5>
+<strong>{{__global.example}}</strong>

```

Comment:
s/example/examples/

Reply:
forgot to leave it out for this PR, done.